stories/ui/Rating.js

```diff
@@ -0,0 +1,153 @@
+/* @flow */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Rating from 'ui/components/Rating';
+import type { Props as RatingProps } from 'ui/components/Rating';
+
+import { createChapters } from '../utils';
+import Provider from '../setup/Provider';
+
+// Some extra styles for the Rating component
+// are included in the additional files.
+import 'amo/components/RatingManager/styles.scss';
```

Comment:
I don't think including these extra styles are necessary. Do you? If there are considerable visual differences then we should just set up storybooks for each one of them (not as part of this issue, though). As a first pass, we should just get a storybook for `Rating` in isolation.

Reply:
I guess I was trying to match as closely as possible. But you're right in this case it's not maybe not too big of a difference.